include/xtensor/xpad.hpp

```diff
@@ -117,6 +118,18 @@ namespace xt
                         sv_end[axis] = xt::range(out.shape()[axis], out.shape()[axis]-ne-1, -1);
                     }
                 }
+                else if (mode == pad_mode::symmetric_101) {
+                    XTENSOR_ASSERT(out.shape()[axis]>1);
```

Comment:
In our style convention there should be white space around comparison or math operators. 

```suggestion
                    XTENSOR_ASSERT(out.shape()[axis] > 1);
```